### ISSUE TITLE
Examples containing extra packages can't be compiled via LaTeX online

### ISSUE THREAD

**Issue by stone-zeng** [learnlatex/learnlatex.github.io#31]
In the `extra-01` page, some examples can't be compiled via LaTeX online. For example, the `xskak` gives

```tex
pdflatex -interaction nonstopmode -recorder -output-directory latex.out /tmp/downloads/tmp_4240/main.tex
/tmp/downloads/tmp_4240/main.tex:4: error: File `xskak.sty' not found
      at <read *>
    from \begin{document}
               ^
/tmp/downloads/tmp_4240/main.tex: Fatal error (no output file produced)
There were errors; /tmp/storage/compilation_2625/output.pdf not updated
```

Examples `tikz`, `pgfplots`, `musixtex` are similar.

**Comment by davidcarlisle** [comment]
@stone-zeng  yes I added a comment in the source perhaps I should have made it visible text.
There is a test version of latexonline using texlive 2020 and I've confirmed the  examples work with that. I don't think the maintainer wants that version widely published yet so the version here is still using texlive 2017 (which is already an from the texlive originally available)

I expect that the texlive 2020 version will be available before we really announce the site.
